src/models/APISupport.ts

```diff
@@ -30,6 +32,8 @@ export class APISupport {
         let header = await this.authContext.armTokenHeader();
         let response = await FetchUtil.httpClientRetries(armEndpoint.toString(),header,'get',3,"");
         let resource_name: string | undefined = core.getInput('loadTestResource');
+        this.resourceName = resource_name;
+        this.subName = this.authContext.subscriptionId;
```

Comment:
Earlier it was sub name. Are we fine with changing to sub id?

Reply:
will change it.